src/reply.rs

```diff
@@ -849,12 +849,16 @@ mod test {
     }
 
     struct AssertSender {
-        expected: Vec<Vec<u8>>,
+        expected: Vec<u8>,
     }
 
     impl super::ReplySender for AssertSender {
         fn send(&self, data: &[&[u8]]) {
-            assert_eq!(self.expected, data);
+            let mut v = vec![];
+            for x in bufs {
```

Comment:
I think there's a typo here. `bufs` isn't defined. Should be `data` I think?

Reply:
Fixed in #137.